tools/storage-usage-delta-calculator/README.md

```diff
@@ -0,0 +1,6 @@
+# Storage delta calculator
+
+A small tool to compare the actual storage use with the one saved within the state.
+Useful as a sanity check if we do any updates (as storage has impact on the amount of tokens that we lock in people's accounts).
+
+WARNING: This was built as a one-time tool - to debug the issue during migration in May 2021 - and is no longer maintained.
```

Comment:
Should we just drop it? Every binary we have in the repo adds substantial amount of linking time.

Reply:
This is a great question - let's discuss it on Zulip. 

From my side, I'm all for removing old tools (or at least disabling their compilation and/or moving to separate place)